src/dynamic-plugin-sdk.ts

```diff
@@ -0,0 +1,554 @@
+/* eslint-disable */
+import * as React from 'react';
+import * as _ from 'lodash';
+
```

Comment:
@sawood14012 Hey, can we not have this codecov thing popup on every line in the PR? Its really distracting while doing the review.

Reply:
you mean the codcov bot ? i guess it is added by openshift org you can configure it in the repo settings telling it to only post once .

Reply:
@rohitkrai03 you can hide them using the `Show annotations` action:
![image](https://user-images.githubusercontent.com/14068621/146584246-01261ebb-601a-4903-8864-6b5a2b1d2be4.png)